Services/AccessControl/classes/class.ilObjRoleGUI.php

```diff
@@ -325,7 +325,7 @@ protected function readRoleProperties(ilObjRole $role)
 
         $rbacreview = $DIC['rbacreview'];
         
-        $data['title'] = $role->getTitle();
+        $data['title'] = $role->_getTranslation($role->getTitle());
```

Comment:
Please change this to a static call, since the method is static: ilObjRole::_getTranslation(...)